kasa/smartprotocolerrors.py

```diff
@@ -0,0 +1,86 @@
+"""Module for defining SMART protocol errors."""
+from enum import Enum
+
+
+class ErrorCode(Enum):
+    """Enum for SMART Error Codes."""
+
+    SUCCESS = 0
+
+    UNKNOWN_ERROR_CODE = 999999
+    ERROR_CODE_NONE = 999998
+    ERROR_CODE_INVALID = 999997
```

Comment:
Have you seen these live / elsewhere? I'm just wondering whether we should add at least this many  "unverified" error codes yet.

Reply:
I created these 3 myself to handle `_mising_()` but as that is now removed from your other comment these 3 are now removed.